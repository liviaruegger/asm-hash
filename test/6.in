Foi na esquina da rua que ele notou o primeiro indicio de que algo estranho ocorria - um gato lia um mapa. Por um instante o Sr. Dursley nao percebeu o que vira - em seguida virou rapidamente a cabeca para dar uma segunda olhada. Havia um gato de listras amarelas sentado na esquina da rua dos Alfeneiros, mas nao havia nenhum mapa a vista. Em que estaria pensando naquela hora? Devia ter sido um efeito da luz. Ele piscou e arregalou os olhos para o gato. O gato o encarou. Enquanto virava a esquina e subia a rua, espiou o gato pelo espelho retrovisor. Ele agora estava lendo a placa que dizia rua dos Alfeneiros - nao, estava olhando a placa: gatos nao podiam ler mapas nem placas. O Sr. Dursley sacudiu a cabeca e tirou o gato do pensamento. Durante o caminho para a cidade elenao pensou em mais nada exceto no grande pedido de brocas que tinha esperancas de receber naquele dia.
