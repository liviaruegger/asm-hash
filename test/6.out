896a06f28a29a4352544dc231913740a
